.circleci/config.yml

```diff
@@ -7,6 +7,15 @@ references:
         user: builder
     working_directory: ~/aeternity
 
+  container2_config: &container2_config
```

Comment:
This approach follows https://github.com/aeternity/aeternity/pull/2088 so this may need update once 2088 commented/rejected/merged.